res/values/setup.xml

```diff
@@ -24,7 +24,8 @@
     
     <!-- Flags to enable/disable some features -->
     <string name = "send_files_to_other_apps">on</string>
-    <string name = "share_feature">on</string>
+    <string name = "share_via_link_feature">on</string>
+    <string name = "share_with_users_feature">on</string>
```

Comment:
Flag values could be treated directly as boolean instead of parsing the string to get a boolean value.